apps/i18n/studio/en_us.json

```diff
@@ -52,6 +56,8 @@
   "catProcedures": "Functions",
   "catText": "Text",
   "catVariables": "Variables",
+  "caveboy": "cave boy",
+  "cavegirl": "cave girl (Jasmine)",
```

Comment:
I know that only some of them currently have canonical names, I assume that's why. But it definitely makes less sense in a context where we expect to have multiple of the same sprite on the screen; I'd leave the names out of this implementation.